blocks/property-search-bar/property-search-bar.css

```diff
@@ -629,7 +684,7 @@ main .section .property-search-bar .btn.btn-secondary:hover {
     display: none;
 }
 
-.property-search-bar.block .no-scroll {
+.no-scroll {
```

Comment:
This shouldn't be in the block level CSS, or it needs to be more qualified.

Reply:
updated

Reply:
Ah i see now.